### PR TITLE
Add block list and update warning

### LINKED ISSUE #49: Full list of blocks and block_ids should be available in the docs.
We should have a full list of blocks and there platform independent definitions (block_ids) listed in the PyTNG docs somewhere.

### PR DESCRIPTION
**Pull request by hmacdope**
I have added a list of all the available blocks and updated the do not use warning slightly.
Closes #49

### CODE REVIEW

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thank you.

I think at this stage you should start splitting the docs into landing (index), installation (TODO), examples, API (page for code and page for blocks etc). 

I would also break the block id page into subheadings and perhaps even convert the code to tables.

File: docs/index.rst [hunk 1]
```diff
@@ -202,13 +202,14 @@ the calling code. However, as the trajectory reading itself is done
...
 
-The success of the last call to ``get_blockid``, ``get_positions``, ``get_box`` and
+The success of the last call to ``get_blockid()``, ``get_positions()``, ``get_box()`` and
 related methods can be checked using the :class:`TNGCurrentIntegratorStep.read_success`
 property, which returns ``True`` if the attempt to read was successful or ``False`` if not.
 Success (``True``) indicates that there was data of the requested block type present at the current step,
 while failure (``False``) indicates either no data present, file corruption or a file reading exception not caught by PyTNG.
 These types of data reading failures are treated on an equal footing as they are in the TNG library.
 
+
```
[the remark this answers is not in the document]
Comment (hmacdope) [reply docs/index.rst]: Done :)

**Review by orbeckst — APPROVED** [review]
Great!! I assume you built the docs locally (`python setup.py build_sphinx`) and looked at it and it's all fine?

**Comment by hmacdope** [comment]
Ahh, docs building fine with Makefile but not with `build_sphinx`. I will investigate. In the meantime do not merge.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[build_sphinx]
+all-files = 1
+source-dir = docs
+build-dir = docs/_build/
+warning-is-error = 0
```
Comment (orbeckst) [inline setup.cfg:5]: This should be 1 for production use.

Fix sphinx until everything works.

File: docs/conf.py [hunk 3]
```diff
@@ -179,12 +181,12 @@
...
 # base URL from which the finished HTML is served.
-html_use_opensearch = 'https://www.mdanalysis.org/RotamerConvolveMD'
+html_use_opensearch = 'https://www.mdanalysis.org/pytng'
 
 # sphinx-sitemap config https://github.com/jdillard/sphinx-sitemap
 #html_baseurl = 'https://www.mdanalysis.org/RotamerConvolveMD/'
-site_url = 'https://www.mdanalysis.org/RotamerConvolveMD/'
-html_extra_path = ['robots.txt']
+site_url = 'https://www.mdanalysis.org/pytng/'
+#html_extra_path = ['robots.txt']
```
Comment (orbeckst) [inline docs/conf.py:189]: Add `robots.txt`
```
User-agent: *

Sitemap: https://www.mdanalysis.org/pytng/sitemap.xml
```

File: setup.cfg (hunk 2 above)
Comment (hmacdope) [reply setup.cfg:5]: Noted :)

**Comment by hmacdope** [comment]
I think ready to go, but you should probably check over again @orbeckst.

**Comment by hmacdope** [comment]
I can merge ?

**Comment by orbeckst** [comment]
If it builds its good. 

Can you raise an issue for adding a test to travis to check that the docs can be built?

**Comment by hmacdope** [comment]
Issue raised in #58

**Pull request merged**